### PR TITLE
Make updates in preparation for CRAN

### PR DESCRIPTION
**Pull request by delosh653**
Going through the package and making updates to submit to CRAN.

### CODE REVIEW

**Comment by dchud** [comment]
I *think* I got all the source/ref data files compressed correctly. Some remaining stuff:

- [ ] handle note/warning on R.utils import in DESCRIPTION (added due to data.table requirement for `fread(gzipped_file)`, but didn't add correctly)
- [ ] update `NEWS.md`
- [ ] create a [release](https://github.com/carriedaymont/growthcleanr/releases) using new bits from `NEWS.md` :)

File: DESCRIPTION [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
+    person("De los Santos","Hannah", email = "hdelossantos@mitre.org", role = c("ctb","cre")),
+    person("Cao","Lusha", role = "ctb"),
+    person("Silva","Steffani", role = "ctb"),
+    person("Zhang","Hanzhe", role = "ctb"),
+    person("Boyas","Matt", role = "ctb"),
+    person("Freedman","David", role = "ctb")
   )
-Maintainer: Robert Grundmeier <grundmeier@email.chop.edu>
-Description: growthcleanr cleans growth data that may contain implausible data based on unit or data range.
+Description: ‘growthcleanr’ identifies implausible anthropometric (e.g., height,
```
Comment (molivier-314) [inline DESCRIPTION:18]: Not sure if growthcleanr is meant to be in quotes.

File: R/extdata.R [hunk 2]
```diff
@@ -186,9 +186,22 @@ NULL
...
 NULL
+
+#' CDC BMI reference data
+#'
+#' Used for extended BMIz computation
+#'
+#' @name CDCref_d
+#'
+#' @section CDCref_d.csv.gz:
+#'
```
Comment (molivier-314) [inline R/extdata.R:203]: Not entirely sure what this is for.

File: inst/extdata/who_ht_maxvel_3sd.csv [hunk 3]
```diff
@@ -1,195 +0,0 @@
-sex,whoagegrp_ht,max_whoinc_1_ht,max_whoinc_2_ht,max_whoinc_3_ht,max_whoinc_4_ht,max_whoinc_6_ht
-0,0,11.9,11.9,15.4,18.2,22.7
```
Comment (molivier-314) [inline inst/extdata/who_ht_maxvel_3sd.csv:2]: Not sure why all these entries are being deleted.

**Review by molivier-314 — APPROVED** [review]

**Pull request merged**